src/widget/weffectparameternamebase.cpp

```diff
@@ -9,12 +9,17 @@
 
 namespace {
 const QString kMimeTextDelimiter = QStringLiteral("\n");
+// for rounding the value display to N decimals
+const int kValDecimals = pow(10, 2);
```

Comment:
Can this become a constexpr? If not use the result as constexpr to avoid runtime overload.

Reply:
meeh, pow() is not constexpr.
Instead of elaborate workarounds like
```
constexpr double kValDecimals() {
    return pow(10, 2);
}
```
I'll go with `constexpr int kValDecimals = 100;`